generators/client/templates/vue/src/main/webapp/app/shared/config/config.ts.ejs

```diff
@@ -5,39 +5,40 @@ import <%=jhiPrefixCapitalized%>Formatter from './formatter';
 <%_ } _%>
 import { setupAxiosInterceptors } from '@/shared/config/axios-interceptor';
 
-import { library } from '@fortawesome/fontawesome-svg-core';
```

Comment:
This import has disappeared :)

Reply:
Sorry, I deleted my fork by mistake, fixing in a new branch